### PR TITLE
add ID to target label

### LINKED ISSUE #30: Add VM ID to target labels
Target labels should contain the VM ID as provided by VCD.

E.g.:  `"__meta_vcd_id": "urn:vcloud:vm:807b799e-c72f-4592-9acd-ccefefe92720"`

### PR DESCRIPTION
**Pull request by dominikhN**
Fixes #30

### CODE REVIEW

File: changelog/unreleased/vm-id-label.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Change: Add VM ID to labels
```
Comment (tboerger) [inline changelog/unreleased/vm-id-label.md:1]: Maybe you change it a little bit? The last line always got to container links to reference like an issue or pull request.

```
Change: Add new label for virtual machine ID

We've added a new label to get the current ID of a virtual machine in the format provided by vCloud Director in the form of `urn:vcloud:vm:807b799e-c72f-4592-9acd-ccefefe92720`.

https://github.com/promhippie/prometheus-vcd-sd/issues/30
```

File: pkg/action/discoverer.go [hunk 2]
```diff
@@ -173,6 +174,7 @@ func (d *Discoverer) getTargets(ctx context.Context) ([]*targetgroup.Group, erro
 						"project", project,
 						"vapp", vappName,
 						"server", server.Name,
+						"ID", server.ID,
```
Comment (tboerger) [inline pkg/action/discoverer.go:177]: The log labels are all lowercase, so let's keep that :)

File: pkg/action/discoverer.go [hunk 3]
```diff
@@ -190,6 +192,7 @@ func (d *Discoverer) getTargets(ctx context.Context) ([]*targetgroup.Group, erro
...
```
Comment (tboerger) [inline pkg/action/discoverer.go:195]: Same here

**Review by tboerger — COMMENTED** [review]
Some smaller comments, after that I'm happy to merge the contribution.

File: pkg/action/discoverer.go (hunk 3 above)
Comment (dominikhN) [reply pkg/action/discoverer.go]: applied in [8bb507d](https://github.com/promhippie/prometheus-vcd-sd/pull/31/commits/8bb507df716907a7279d67e92d7664cf918b5fd4)

File: pkg/action/discoverer.go (hunk 2 above)
Comment (dominikhN) [reply pkg/action/discoverer.go]: applied in [8bb507d](https://github.com/promhippie/prometheus-vcd-sd/pull/31/commits/8bb507df716907a7279d67e92d7664cf918b5fd4)

File: changelog/unreleased/vm-id-label.md (hunk 1 above)
Comment (dominikhN) [reply changelog/unreleased/vm-id-label.md]: applied in [8bb507d](https://github.com/promhippie/prometheus-vcd-sd/pull/31/commits/8bb507df716907a7279d67e92d7664cf918b5fd4)

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
The build fails for an unrelated reason, I need to fix that in a followup pr. Thanks for your work.

**Pull request merged**